student.rb

```diff
@@ -1,12 +1,22 @@
-require_relative 'Person'
+require_relative './person'
 
 class Student < Person
-  def initialize(id, age, classroom, name: 'Unknown', parent_permission: true)
-    super(id, name, age, parent_permission)
+  attr_reader :classroom
+
+  def initialize(age, classroom, name: 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
     @classroom = classroom
   end
```

Comment:
### Since you didn't make changes to your `Person Class` I don't have access to it that's why I am making the comment here 👍 
- [ ] In your `Person class`, I noticed that you've haven't defined the rentals array to store the associated rentals. So kindly implement this. 💯 
- After which to fully meet the requirements, you need to implement the logic to associate Rental instances with Person objects. 👍 

- [ ] I recommend adding a method called `add_rental` in your Person class that allows you to associate rentals with a person 😉 
- By implementing this logic, you'll fully establish the `has-many relationship` between `Person` and `Rental`, fulfilling the requirements of the project 💯 
- [ ] Lastly, please ensure that `Getters` and `Setters` for @rentals should be present 👍 
- Project Requirement

> Create the many-to-many (also has-many-through) relationship between Person and Book using the intermediate class Rental. The following should be implemented:
> Create the has-many side of Person and Rental (a person has many rentals).